templates/user/register.html.twig

```diff
@@ -0,0 +1,7 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Register{% endblock %}
+
+{% block body %}
+    {{ form(form) }}
```

Comment:
add an `h1`?

We should probably have one on each page

Reply:
Yep, probably add in the layout via the title block?